### DISCUSSION TITLE
Apple silicon support

### DISCUSSION THREAD

**Discussion by tiborpino** [Eloston/ungoogled-chromium#1839] [Q&A]
Hi there. Do you plan support for Apple silicon? I'm using intel based version from homebrew but it would be great to have an option to install apple silicon binary. Thank you.

**Comment by networkException** [comment]
See https://github.com/ungoogled-software/ungoogled-chromium-macos/issues/69

**Reply by tiborpino** [reply]
But I don't want it to build on my own. I would like to install it and update via homebrew...

**Reply by kramred** [reply]
I do not fully understand why you're not getting the arm64 version via `brew` - did you check the output of `chrome://version/`?

When you check the [formula](https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/eloston-chromium.rb) for ungoogled-chromium that `brew` uses you'll see that it has both the `x86_64` and `arm64` versions that you can also find in the [releases built by GitHub Actions](https://github.com/kramred/ungoogled-chromium-macos/releases) of my repo, which `brew` mostly uses for the binaries.

So, if your version of `brew` is up to date you should be getting the `arm64` version, i.e. the native binary for Apple silicon.
